go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/jenkins-x/jx-cdevents-adapter
+
+go 1.19
+
+require (
+	github.com/gorilla/mux v1.8.0 // indirect
+	github.com/sirupsen/logrus v1.9.0 // indirect
```

Comment:
NIT (Ignore if it's too much work or can be done in a subsequent PR): Since we are starting fresh, may be use [zap](https://github.com/uber-go/zap)? Logrus is in maintenance mode, and zap is just so much better (kubebuilder v3 already creates a project with zap for example). In the future, I do want jx to move away from logrus to zap :)